FaissMask/FaissMask.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
Since this is a library and is not making use of any .NET 9 features specifically, I would prefer to keep `TargetFramework` as-is for now as this change would require an application to upgrade to .NET 9 before they could use the new version. No concerns with changing the `TargetFramework` for the test project or the example projects, but just want to keep this one as-is.